spec/rspec/support/object_formatter_spec.rb

```diff
@@ -158,6 +158,13 @@ def with_delegate_loaded
             matcher_without_a_description.inspect)
         end
       end
+
+      context 'with truncation' do
+        it 'produces an output of limited length' do
+          formatter = ObjectFormatter.new(10)
```

Comment:
ah no, nil is fine. the -1 is less good than nil. Can you add a test for that case though? Also shorter inspect strings?